Sources/DataDomeApollo/DataDomeInterceptorProvider.swift

```diff
@@ -63,7 +64,7 @@ public class DataDomeInterceptorProvider: InterceptorProvider {
         } else {
             interceptors.append(contentsOf: [
                 ResponseCodeInterceptor(),
-                JSONResponseParsingInterceptor(cacheKeyForObject: self.store.cacheKeyForObject),
```

Comment:
Neither `JSONResponseParsingInterceptor` has an initialiser that accepts `cacheKeyForObject` no more, nor `ApolloStore` has `cacheKeyForObject` as a member.